package.json

```diff
@@ -71,9 +71,12 @@
   "repository": "forcedotcom/salesforcedx-vscode",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "node": "12.22.9",
+    "node": "^14.0.0",
     "npm": "^6.9.0"
```

Comment:
Can we also update npm version to [6.14.11](https://nodejs.org/en/download/releases/) to match the node version